components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/admin/v1/impl/ThrottlingApiServiceImpl.java

```diff
@@ -523,6 +535,84 @@ public Response throttlingPoliciesSubscriptionPost(String contentType, Subscript
         return null;
     }
 
+    /**
+     * Returns list of throttling policy details filtered using query parameters
+     *
+     * @param query filtering parameters
+     * @return Retrieves Throttle Policies List
+     */
+    @Override
+    public Response throttlingPolicySearch(String query, MessageContext messageContext)
+            throws APIManagementException {
+        ThrottlePolicyDetailsListDTO resultListDTO = new ThrottlePolicyDetailsListDTO();
+        String policyType;
+        Map<String, String> filters;
+        if (query == null) {
+            query = "type:" + ALL_TYPES;
+        } else if (query.toLowerCase().indexOf("type:") != 0) {
+            String errorMessage = "Invalid query format";
+            RestApiUtil.handleInternalServerError(errorMessage, log);
+        }
+        if (log.isDebugEnabled()) {
```

Comment:
log.isDebugEnabled() check is not required when logging a single string. Only if there are string concatanations in the argument, we need to have that check.

Reply:
Fixed with 30d0ecb9c29c0024fa182ba597c8eda4c128c451